src/courseware/course/sequence/Unit/hooks/useExamAccess.test.jsx

```diff
@@ -16,10 +17,27 @@ jest.mock('@edx/frontend-lib-special-exams', () => ({
 
 const id = 'test-id';
 
+// This object allows us to manipulate the value of the accessToken.
+const testAccessToken = { curr: '' };
+
 const mockFetchExamAccessToken = jest.fn().mockImplementation(() => Promise.resolve());
 useFetchExamAccessToken.mockReturnValue(mockFetchExamAccessToken);
 
-const testAccessToken = 'test-access-token';
+const mockUseIsExam = (initialState = false) => {
+  const [isExam, setIsExam] = useState(initialState);
+
+  // This setTimeout block is intended to replicate the case where a unit is an exam, but
+  // the call to fetch exam metada has not yet completed. That is the value of isExam starts
```

Comment:
Is this supposed to be metadata?
```suggestion
  // the call to fetch exam metadata has not yet completed. That is the value of isExam starts
```

Reply:
Thanks for catching that! I'll push up a fix.